frontend/gpay_merchant/lib/maptry.dart

```diff
@@ -0,0 +1,98 @@
+import 'dart:async';
+import 'package:geocoder/geocoder.dart';
+import 'package:flutter/material.dart';
+import 'package:google_fonts/google_fonts.dart';
+import 'package:google_maps_flutter/google_maps_flutter.dart';
+import 'package:google_sign_in/google_sign_in.dart';
+
+class MapSample extends StatefulWidget {
+  MapSample({Key key, this.street, this.city, this.latitude, this.longitude}) : super(key: key);
+  String street;
+  String city;
+  double latitude;
+  double longitude;
+  @override
+  State<MapSample> createState() => MapSampleState();
+}
+
+class MapSampleState extends State<MapSample> {
+  Completer<GoogleMapController> _controller = Completer();
```

Comment:
variables names should be more descriptive, eg. _googleMapController

This way, when I later encounter this variable, it is clear what it holds.

Reply:
@trblessy I have made all the changes you requested